pkgs/development/compilers/gerbil/unstable.nix

```diff
@@ -1,13 +1,14 @@
-{ stdenv, callPackage, fetchgit, gambit-unstable }:
+{ stdenv, callPackage, fetchFromGitHub, gambit-unstable }:
 
 callPackage ./build.nix {
-  version = "unstable-2018-09-06";
-  git-version = "0.14-DEV";
+  version = "unstable-2018-11-19";
+  git-version = "0.15-DEV";
   GAMBIT = gambit-unstable;
-  SRC = fetchgit {
-    url = "https://github.com/vyzo/gerbil.git";
-    rev = "184cb635c82517d5d75d7966dcdf1d25ad863dac";
-    sha256 = "1ljzbpc36i9zpzfwra5hpfbgzj1dyzzp50h5jf976n8qr9x4l7an";
+  SRC = fetchFromGitHub {
+    owner = "vyzo";
+    repo = "gerbil";
+    rev = "11819599207c718ed50ac6f22bfa006a6c33ff4b";
```

Comment:
```suggestion
    rev = git-version;
```

Reply:
In general the revision does not correspond to an exact namable git version.